moped-editor/src/views/projects/projectView/ProjectComponents/ProjectComponentsList.js

```diff
@@ -1,10 +1,11 @@
 import ComponentListItem from "./ComponentListItem";
-import Button from "@mui/material/Button";
+import IconButton from "@mui/material/IconButton";
```

Comment:
thanks for also updating these. i know we have an issue to address this, but chipping away at them in other issues is ✨